src/holochainClient.js

```diff
@@ -45,6 +45,7 @@ async function initAndGetHolochainClient () {
     if (HOLOCHAIN_LOGGING) {
       console.log('🎉 Successfully connected to Holochain!')
     }
+    return holochainClient
```

Comment:
so confused that this is being added now. I don't understand how this function was working without this line.
 /shrug

Reply:
:100: :joy_cat: